common/internal_model/src/test/scala/uk/ac/wellcome/models/work/generators/InstantGenerators.scala

```diff
@@ -0,0 +1,36 @@
+package uk.ac.wellcome.models.work.generators
+
+import java.time.{Instant, LocalDateTime}
+
+import uk.ac.wellcome.models.work.internal._
+
+import scala.concurrent.duration._
+import scala.util.Random
+
+trait InstantGenerators {
```

Comment:
Really like this trait. I should probably backport it to the transformer tests cause this is much nicer than what I did there